src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -61,8 +72,11 @@
 
     @Test
     public void getStatisticAboutBanana() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (FileNotFoundException e) {
```

Comment:
Catching `FileNotFoundException` is not appropriate here because the `getStatistic` method can throw `IOException`. Consider catching `IOException` instead.